tests/acceptance/features/apiSpaces/tusUpload.feature

```diff
@@ -0,0 +1,27 @@
+@api @skipOnOcV10
+Feature: upload resources using TUS protocol
+  As a user
+  I want to be able to upload files
+  So that I can store and share files between multiple client systems
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+
+
+  Scenario: upload a file within the set quota to a project space 
+    Given user "Alice" has created a space "Project Jupiter" of type "project" with quota "10000"
+    When user "Alice" uploads a file "qrcode.png" via TUS inside of the space "Project Jupiter" using the WebDAV API
+    Then the HTTP status code should be "204"
```

Comment:
It's hard to know from reading the test how big the file is. I would prefer to upload a text file with a given content, or to create a file with a size defined in the step